ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigDeneb.java

```diff
@@ -52,6 +52,11 @@ static SpecConfigDeneb required(final SpecConfig specConfig) {
 
   boolean isKZGNoop();
 
+  int getEpochsStoreBlobs();
+
+  /** Maybe user input, use {@link #getEpochsStoreBlobs()} instead */
+  Optional<Integer> getMaybeEpochsStoreBlobs();
```

Comment:
We don't need to expose that method though right, can just store the optional variable in the SpecConfigDenebImpl class and use it in `getEpochsStoreBlobs`

Reply:
I've added it in case next fork could change minEpochs spec value, in this case `getEpochsStoreBlobs` may need to be changed. Actually it's over-complication at this stage and I'm ok to remove it for now